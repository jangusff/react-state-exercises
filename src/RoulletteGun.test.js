import React from 'react';
import ReactDOM from 'react-dom'
import renderer from 'react-test-renderer'
import RouletteGun from './RoulletteGun'

describe(`RouletteGun Component`, () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<RouletteGun bulletInChamber={4} />, div);
    ReactDOM.unmountComponentAtNode(div);
  })
  
  //it.skip('renders the UI as expected', () => {
  it('renders the UI as expected', () => {
    expect(
      renderer.create(<RouletteGun bulletInChamber={4} />).toJSON()
    ).toMatchSnapshot()
  })
})