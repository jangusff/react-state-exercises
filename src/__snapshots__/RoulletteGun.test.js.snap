// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RouletteGun Component renders the UI as expected 1`] = `
<div
  className="RouletteGun"
>
  <button
    onClick={[Function]}
  >
    Pull the trigger!
  </button>
  <br />
  <p>
    You're safe...for now!
  </p>
</div>
`;
